helm-chart/renku-gateway/templates/configmap.yaml

```diff
@@ -201,6 +201,11 @@ data:
           trustForwardHeader = true
           authResponseHeaders = ["Authorization", "Renku-user-id", "Renku-user-fullname", "Renku-user-email"]
 
+        [http.middlewares.auth-notebook.forwardauth]
+          address = "http://{{ template "gateway.fullname" . }}-auth/?auth=notebook"
+          trustForwardHeader = true
+          authResponseHeaders = ["Renku-gitlab-accesstoken", "Renku-user-id", "Renku-user-fullname", "Renku-user-email"]
```

Comment:
@ableuler I see all of `"Renku-gitlab-accesstoken", "Renku-user-id", "Renku-user-fullname", "Renku-user-email"` in the header inside the requests in the notebooks service. However I do not see just the regular `Renku-User` header. Was this also supposed to be included here? If I include `Renku-User` in the list of headers here it does show up in notebooks.

Reply:
There's a config to determine which headers will actually taken over from the response of the middleware: https://github.com/SwissDataScienceCenter/renku-gateway/blob/429a4004fc91badabef6fbe83a5151426f6bc045/helm-chart/renku-gateway/templates/configmap.yaml#L207

Reply:
So you'd also have to add `Renku-User` in this list in order to receive it in the notebooks service.